src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -162,6 +162,27 @@ def tag_resource(tags, resource_group_name=None, resource_provider_namespace=Non
                          resource_id, api_version)
     return res.tag(tags)
 
+def get_deployment_operations(client, resource_group_name, deployment_name, operation_id=None,
+                              operation_ids=None):
+    """Gets a deployments operation.
+    :param resource_group_name: The name of the resource group. The name
+     is case insensitive.
+    :type resource_group_name: str
+    :param deployment_name: The name of the deployment.
+    :type deployment_name: str
+    :param operation_ids: A comma separated list of ids to get.
+    :type operation_ids: str
+    """
```

Comment:
I don't think we need the doc-string here for individual arguments, since the argument names are pretty clear. Also all the help content are already exposed in `_params.py`.  `Gets a deployments operation` can stay for command's short summary, but i suggest update the wording to `get a deployment's operations`? Once it is done, please double check using '-h'